skin/popup.css

```diff
@@ -377,3 +377,12 @@ font-size: 16px;
 .hidden {
   display: none;
 }
+p {
+  font-size: 12px;
+}
+span {
+  font-size: 12px;
+}
+#overlay label {
+  font-size: 13px;
```

Comment:
These are all too broadly scoped. You need to scope the font size change specifically to the buttons, this makes the text for the entire plugin very tiny.